release/models/lacp/openconfig-lacp.yang

```diff
@@ -234,6 +250,18 @@ module openconfig-lacp {
         port";
     }
 
+    leaf port-priority {
+      type uint16;
+      description
+        "Member interface's priority in its aggregate interface.";
+    }
+
+    leaf partner-port-priority {
+      type uint16;
+      description
+        "Member interface partner's priority in its aggregate interface.";
+    }
+
```

Comment:
Remove these leafs.  

(Note: Instead of explicitly adding these leaves here, the preferred style is to include the 'config' grouping in your state container.  This helps ensure all config leaves have equivalent state leaves)

Reply:
Used config grouping with `interface` and `port-priority` for both `config` and `state`. But kept `partner-port-priority` under state as it is not a configurable parameter.